### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by Aut0R3V**

### CODE REVIEW

**Comment by afbjorklund** [comment]
It would be good if the commit referenced _why_ something was changed

Which files that were modified is already apparent from the patch itself.

And branch name "patch-1" doesn't really help much to clarify either ?

Please also link this PR, to the issue: #9803

**Comment by Aut0R3V** [comment]
@afbjorklund, apologies. I'll update these details and send a new PR. Closing this for now

**Comment by afbjorklund** [comment]
> @afbjorklund, apologies. I'll update these details and send a new PR. Closing this for now

You don't really have the close the branch (the name it's not _that_ important), and you can --amend the commit message

Thanks for trying to improve the documentation!

**Comment by Aut0R3V** [comment]
Okay,

**Comment by Aut0R3V** [comment]
Okay, I'll update the changes. Thanks

**Comment by afbjorklund** [comment]
Some of these instructions can be a bit sensitive and it could take a while to get the wording "just right".

So the review might take some time... One idea I had was that maybe we should try do draw a diagram.

Something simple, like how they were explaining docker engine vs. docker machine ?

https://docs.docker.com/machine/#whats-the-difference-between-docker-engine-and-docker-machine

But I sort of promised this also for the networking, and didn't really follow through.

----

Another thing we struggle with is the balance between "easy-to-read" and "detailed" (or "correct")

So we might have to do a TL;DR summary at the top, and then provide more details further down.

**Comment by Aut0R3V** [comment]
I agree, the documentation would be better by that segregation. Can you recommend any interface for the diagrammatic representation?

**Comment by Aut0R3V** [comment]
I agree, the documentation would be better by that segregation. I'll update the PR.

**Comment by afbjorklund** [comment]
Another thing that could be good to know, is that even though we currently rely on particular technology like VirtualBox or Docker we ultimately want to make it more agnostic and talk about a more generic "hypervisor" and "runtime" with more options available.
We are also moving away from Docker Machine, since it is deprecated upstream and not supported anymore... So even though nothing changes in the _current_ implementation, it would be nice to keep it more open-ended and not rely on particular vendors.

Again this is overwhelming for the user that is new to all of it, but good to have in mind while writing the documentation (perhaps).

File: site/content/en/docs/drivers/_index.md [hunk 1]
```diff
@@ -8,36 +8,28 @@ description: >
 aliases:
   - /docs/reference/drivers
 ---
-Added a one liner description of bare metal environments.
+
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:11]: Added a one liner description of bare metal environments.

File: site/content/en/docs/drivers/_index.md [hunk 2]
```diff
@@ -8,36 +8,28 @@ description: >
...
 There are various options through which minikube can be deployed. These include using a VM, a container, or bare metal. In the case of bare metal environments, a server can be used as a single tenant. 
 
-Difference between Docker Desktop and Docker Engine.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md]: Difference between Docker Desktop and Docker Engine.

File: site/content/en/docs/drivers/_index.md [hunk 3]
```diff
@@ -8,36 +8,28 @@ description: >
 aliases:
   - /docs/reference/drivers
 ---
-Added a one liner description of bare metal environments.
+
 There are various options through which minikube can be deployed. These include using a VM, a container, or bare metal. In the case of bare metal environments, a server can be used as a single tenant. 
 
-Difference between Docker Desktop and Docker Engine. 
 In the windows operating system, docker is sometimes referred to as Docker Engine. Docker desktop can be used to run containers of both Linux and Windows os.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:15]: Difference between Podman and Docker

File: site/content/en/docs/drivers/_index.md [hunk 4]
```diff
@@ -8,36 +8,28 @@ description: >
...
 ---
-Added a one liner description of bare metal environments.
+
 There are various options through which minikube can be deployed. These include using a VM, a container, or bare metal. In the case of bare metal environments, a server can be used as a single tenant. 
 
-Difference between Docker Desktop and Docker Engine. 
 In the windows operating system, docker is sometimes referred to as Docker Engine. Docker desktop can be used to run containers of both Linux and Windows os. 
 
-Difference between Podman and Docker
 Docker and Podman are container engines and have almost similar commands for their operations. Podman directly interacts with the container content, however, Docker uses daemon for this.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:16]: Difference between Podman and Docker

File: site/content/en/docs/drivers/_index.md [hunk 5]
```diff
@@ -8,9 +8,32 @@ description: >
...
-minikube can be deployed as a VM, a container, or bare-metal.
 
-To do so, we use the [Docker Machine](https://github.com/docker/machine) library to provide a consistent way to interact with different environments. Here is what's supported:
+There are various options through which minikube can be deployed. These include using a VM, a container, or bare metal. In the case of bare metal environments, a server can be used as a single tenant. 
+
+In the windows operating system, docker is sometimes referred to as Docker Engine. Docker desktop can be used to run containers of both Linux and Windows os. 
+
+Docker and Podman are container engines and have almost similar commands for their operations. Podman directly interacts with the container content, however, Docker uses daemon for this.
+
+The [Docker Machine](https://github.com/docker/machine) library provides dynamic ways to interact with different environments. In simple words, it's a VM that runs docker and Linux simultaneously.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:18]: Added a one liner explanation of Docker machine library

File: site/content/en/docs/drivers/_index.md [hunk 6]
```diff
@@ -8,9 +8,32 @@ description: >
...
-To do so, we use the [Docker Machine](https://github.com/docker/machine) library to provide a consistent way to interact with different environments. Here is what's supported:
+There are various options through which minikube can be deployed. These include using a VM, a container, or bare metal. In the case of bare metal environments, a server can be used as a single tenant. 
+
+In the windows operating system, docker is sometimes referred to as Docker Engine. Docker desktop can be used to run containers of both Linux and Windows os. 
+
+Docker and Podman are container engines and have almost similar commands for their operations. Podman directly interacts with the container content, however, Docker uses daemon for this.
+
+The [Docker Machine](https://github.com/docker/machine) library provides dynamic ways to interact with different environments. In simple words, it's a VM that runs docker and Linux simultaneously.   
+
+A hypervisor supervises the creating and running of VMs hence allowing multiple guest VM for a single guest. Through this process, the host is provided access to the resources of the VMs.Different kinds of operating systems need a different kind of hypervisors.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:20]: Explanation about hypervisor and Hyper-V

File: site/content/en/docs/drivers/_index.md [hunk 7]
```diff
@@ -8,9 +8,32 @@ description: >
...
+
+Docker and Podman are container engines and have almost similar commands for their operations. Podman directly interacts with the container content, however, Docker uses daemon for this.
+
+The [Docker Machine](https://github.com/docker/machine) library provides dynamic ways to interact with different environments. In simple words, it's a VM that runs docker and Linux simultaneously.   
+
+A hypervisor supervises the creating and running of VMs hence allowing multiple guest VM for a single guest. Through this process, the host is provided access to the resources of the VMs.Different kinds of operating systems need a different kind of hypervisors. 
+
+Hyper-V provides another alternative of running Linux containers on Windows using docker. In this approach, every Window containers share a single kernel but every Linux container has a separate Linux kernel. 
+
+KVM2 or a VirtualBox can be used as a hypervisor for linux. An alternative container runtime to docker driver is the podman driver.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:24]: Intro to KVM2 and explanation of the use of podman

File: site/content/en/docs/drivers/_index.md [hunk 8]
```diff
@@ -8,9 +8,32 @@ description: >
...
+
+Macbook doesn't directly run based on any containers and a different set of hypervisor are required. HyperKit, VirtualBox, VMware Fusion are some of the possible hypervisors that can be used.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:26]: Explanation of the different possible options for macOS

File: site/content/en/docs/drivers/_index.md [hunk 9]
```diff
@@ -8,9 +8,32 @@ description: >
...
+
+A hypervisor supervises the creating and running of VMs hence allowing multiple guest VM for a single guest. Through this process, the host is provided access to the resources of the VMs.Different kinds of operating systems need a different kind of hypervisors. 
+
+Hyper-V provides another alternative of running Linux containers on Windows using docker. In this approach, every Window containers share a single kernel but every Linux container has a separate Linux kernel. 
+
+KVM2 or a VirtualBox can be used as a hypervisor for linux. An alternative container runtime to docker driver is the podman driver.
+
+Macbook doesn't directly run based on any containers and a different set of hypervisor are required. HyperKit, VirtualBox, VMware Fusion are some of the possible hypervisors that can be used. 
+
+Parallels driver is another driver option for users working with the Parallels Desktop. It provides an easy blending of windows and OS.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:28]: Intro to parallels

File: site/content/en/docs/drivers/_index.md [hunk 10]
```diff
@@ -8,9 +8,32 @@ description: >
...
+Hyper-V provides another alternative of running Linux containers on Windows using docker. In this approach, every Window containers share a single kernel but every Linux container has a separate Linux kernel. 
+
+KVM2 or a VirtualBox can be used as a hypervisor for linux. An alternative container runtime to docker driver is the podman driver.
+
+Macbook doesn't directly run based on any containers and a different set of hypervisor are required. HyperKit, VirtualBox, VMware Fusion are some of the possible hypervisors that can be used. 
+
+Parallels driver is another driver option for users working with the Parallels Desktop. It provides an easy blending of windows and OS.  
+In macbooks, the docker cli is preconfigured to interact with the docker daemon that’s running directly on our macbook.
+
+In the case of a windows system, a docker based native Kubernetes integration can be used. This would be ideal for Windows 10. However, a hypervisor can provide the best performance.
```
Comment (Aut0R3V) [inline site/content/en/docs/drivers/_index.md:31]: A little detail about installation in windows

**Comment by bl-ue** [comment]
So @Aut0R3V you ended up by deleting `_index.md`? :smile:

**Comment by Aut0R3V** [comment]
> So @Aut0R3V you ended up by deleting `_index.md`? 😄

I felt like there were some changes that were to be added. It might've needed more than one commits, so I gave a new pull request.

**Pull request closed without merging**